aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -398,6 +398,14 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 					},
 				},
 			},
+			"bgp_lan_interfaces_count": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      1,
```

Comment:
Should this have a default value if it is only used with certain features: BGP over LAN in Azure?

Reply:
This attribute is only valid for Azure. Setting it to 1 means nothing for other clouds. It will show up in the state file though.